### ISSUE TITLE
Powershell provider - mapping network drive does not work nor produces error output

### ISSUE THREAD

**Issue by ntr** [fsprojects/FSharp.Management#57]
I'm trying to map network drive using powershell provider by executing code in interactive:
```fsharp
type PS = PowerShellProvider< "Microsoft.PowerShell.Management;Microsoft.PowerShell.Core" >
PS.``New-PSDrive``(name="M", pSProvider="FileSystem", root="\\127.0.0.1\c$", persist=true)
```
It fails to create the drive and does not provide any information about the error.

Corresponding powershell creates network drive:
```powershell
New-PSDrive –Name "M" –PSProvider FileSystem –Root "\\127.0.0.1\c$" -persist
```

**Comment by sergey-tihon** [comment]
The reason is that you used incorrect `string` literal for `root` parameter.
Use `@` symbol to convert it to verbatim string literal => `@"\\127.0.0.1\c$"`.

Full source code:
```fsharp
#r "FSharp.Management.PowerShell.dll"
#r "System.Management.Automation.dll"
open FSharp.Management

type PS = PowerShellProvider< "Microsoft.PowerShell.Management;Microsoft.PowerShell.Core" >

PS.``New-PSDrive``(name="M", pSProvider="FileSystem", root= @"\\127.0.0.1\c$", persist=true)
```

**Comment by ntr** [comment]
Yes, that helped. But shouldn't the provider report to user what is the problem? Via returned object/exception?

**Comment by sergey-tihon** [comment]
@ntr you are right, we need to find a way to catch errors and show them to the user

**Comment by sergey-tihon** [comment]
@kcheick Would you like to improve error reporting in `PowerShell` type provider?

**Comment by sergey-tihon** [comment]
Should be fixed in [v0.4.0](https://www.nuget.org/packages/FSharp.Management/0.4.0)
